doc/advanced-usage.rst

```diff
@@ -78,7 +78,7 @@ when the integrity of a machine is not trusted. (https://tools.ietf.org/html/dra
     of logging and access control.  This access control may utilize
     information about the path through which the connection was received (as
     collected with SSH_AGENT_FORWARDING_NOTICE messages; however, the path
-    is reliable only up to and including the first unreliable machine.).
+    is reliable only up to and including the first unreliable machine).
```

Comment:
This is a quote from https://datatracker.ietf.org/doc/html/draft-ietf-secsh-agent-02
I think we should not change quoted text.

Reply:
Fair enough. Let me change that.